packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss

```diff
@@ -18,24 +18,196 @@
 // InlineEdit uses the following Carbon for IBM Products components:
 // TODO: @import(s) of IBM Products component styles used by InlineEdit
 
+// cut down version borrowed from 'carbon-components/scss/globals/scss/vendor/@carbon/styles/scss/utilities/focus-outline';
+@mixin input-outline($color: $focus) {
+  outline: $spacing-01 solid $color;
+  outline-offset: calc(-1 * $spacing-01);
+
+  @media screen and (prefers-contrast) {
+    outline-style: dotted;
+  }
+}
+
+@mixin input-button-defaults() {
+  width: var(--size);
+  height: 100%;
+  min-height: initial;
+  max-height: var(--size);
+
+  // stylelint-disable-next-line carbon/layout-token-use
+  padding: calc(0.5 * (var(--size) - var(--icon-size)));
+  border: 2px solid transparent;
+}
+
 // Define all component styles in a mixin which is then exported using
 // the Carbon import-once mechanism.
 @mixin inline-edit {
   // The block part of our conventional BEM class names (blockClass__E--M).
   $block-class: #{$pkg-prefix}--inline-edit;
 
-  .#{$block-class} {
+  @include block-wrap($block-class, 3) {
+    --size: #{$spacing-08};
+    --icon-size: #{$spacing-05};
+    --background-color: #{$ui-01};
```

Comment:
I think CSS variable names need to be scoped, as they are global once on the DOM. Our usual pattern here is to include the block-class, which seems a good plan, _i.e._
- `--#{$block-class}--size`
- `--#{$block-class}--icon-size`
- `--#{$block-class}--background-color`